src/js/media-poster-image.js

```diff
@@ -0,0 +1,70 @@
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+import { MediaUIAttributes } from './constants.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>
+    :host {
+      height: 100%;
+      width: 100%;
+      position: relative;
+    }
+
+    div {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-position: center;
+      background-size: cover;
+      background-repeat: no-repeat;
+    }
+
+    :host([${MediaUIAttributes.MEDIA_HAS_PLAYED}]) div {
+      display: none;
+    }
+  </style>
+
+  <div part="placeholder"></div>
+  <div part="image"></div>
+`;
```

Comment:
@cjpillsbury 

> This feels like it should go in one of our well-defined slots or have its own slot, and not the "bottom-chrome" slot.

Would something like `slot="centered-chrome"` be specified in the template (we wouldn't want to require the user to specify the slot)

Is that what you had in mind?

Reply:
Also, if we do that, could the user still specify a second `slot="centered-chrome"`, for example:

User wants to use a placeholder and have a play button in the center:

```
<media-controller>
  <media-poster-image></media-poster-image>
  <div slot="centered-chrome">
     <media-play-button></media-play-button>
  </div>
</media-controller>
```

In either case, I want to make sure that use case works, which I imagine will be pretty common

Reply:
The poster element feels like it belongs in the 'basic' example, and I'd love to avoid making people learn the layout slots in basic if we can avoid it. If we want to get more specific with the poster placement in 'advanced', no problem.

Reply:
Correct, I agree with that. The user definitely shouldn't have to specify any slot.

I'm just referring to the template for `media-poster-image` using the `"centered-chrome"` slot under the hood.